.gitignore

```diff
@@ -1,5 +1,9 @@
 target/
 
+# API keys
+src/main/webapp/config.js
```

Comment:
maybe just "config.js" in case we accidentally move it around?

Reply:
will do.